ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/router-general.md

```diff
@@ -75,5 +75,21 @@ router general
       routes dynamic prefix-list DYNAMIC_TEST_PREFIX_LIST_2
       exit
    !
+   control-functions
+      code unit code1
+         function ACCEPT_ALL() {
+           return true;
+           }
+         EOF
+         compile
+         commit
```

Comment:
these two lines are not under the code unit, but at the parent level.

Reply:
Moved compile and commit to parent level.